d2common/d2data/d2datadict/item_affix.go

```diff
@@ -109,11 +107,11 @@ func loadDictionary(
 
 func createItemAffixRecords(
 	d *d2common.DataDictionary,
-	r []*ItemAffixCommonRecord,
```

Comment:
I haven't looked at the details for where memory is allocated in OpenDiablo2, but sometimes passing in an existing slice may help us reuse the underlying backing array, so we don't have to allocate memory multiple times if we invoke `createItemAffixRecords` more than once.

Reply:
Good point. But this fonction is actually called twice (once for magic prefix and once for magic suffix) with around 750 items when the game launches. I don't think the underlying array is ever reused anyway.